test/conformance/runtime/header_test.go

```diff
@@ -91,6 +91,8 @@ func TestShouldHaveHeadersSet(t *testing.T) {
 		// We expect the value to be a 64-bit or 128-bit hex string
 		"x-b3-traceid": regexp.MustCompile("[0-9a-f]{16}|[0-9a-f]{32}"),
 
+		"traceparent": regexp.MustCompile("[0-9]{2}-[0-9a-f]{32}-[0-9a-f]{16}-[0-9]{2}"),
```

Comment:
link to spec?  Seems odd to require this in conformance, do our clients always send this?

Reply:
This change see to it both B3 and W3C headers are populated to the user container.
Spec: https://www.w3.org/TR/trace-context/

Reply:
Also this was proposed to be added by @evankanderson as a validation test for this change.

Reply:
I thought if we didn't have an e2e test, it probably wouldn't keep working

Reply:
Are we always setting the tracing header, or is it sampled in some way? Do we need the client to set the header?

Reply:
Client will set one of B3 or W3C header. We will pick one (W3C is preferred), and propagate both headers with up to date values.